### DISCUSSION TITLE
口罩禁令解除仍不願脫

### DISCUSSION THREAD

**Discussion by ALAN30359704** [1101-GS4719-C/Python-Community-GS4719-C#1236] [General]
戶外、大眾運輸等處[口罩](https://udn.com/search/tagging/2/%E5%8F%A3%E7%BD%A9)令解除，但許多人不習慣仍戴著口罩，台北市立聯合醫院中興院區皮膚科主任潘企岳說，戶外戴口罩有一大好處，就是「[防曬]也可以防空汙，尤其天氣炎熱時，陽光[紫外線]對皮膚是一大傷害，戴口罩可遮蔽三分之二臉部，避免紫外線照射，額頭部位則可用陽傘、寬邊帽遮陽，在人潮稀少或空曠處再脫下口罩通風即可，以降低青春痘機會，其實大家都可以這樣做，「因為，我也是這樣。」

潘企岳說，陽光紫外線會加速皮膚老化，長期照射還有致癌風險，天氣炎熱時，一定要注意防曬，以往臉部防曬為塗防曬乳、撐陽傘、戴寬邊帽等，但塗防曬乳過於油膩，恐會長出青春痘，口罩可直接阻擋陽光紫外線的照射，也可阻擋從地面反射的紫外線，是相當好的防曬工具。另因汽機車廢氣或受氣候影響造成的空氣品質變差，戴口罩也可以避免吸入空氣汙染物質。
但長時間戴口罩，皮膚容易悶熱。潘企岳說，女性平時戴口罩時，應以淡妝為主，避免毛細孔被厚重化妝品阻塞，長出痘痘或引起過敏，返家後再用洗面乳、卸妝油徹底卸妝，並加以保濕，降低粉刺、青春痘的機會。而油性膚質的人，長時間戴口罩，皮膚容易悶熱、出油，造成下巴長出許多青春痘，天氣炎熱時，此類患者人數比一般時候約增二成，建議在人潮稀少或空曠處，油性膚質的人可脫下口罩通風。
潘企岳說，如果皮膚過於悶熱出現粉刺時，一定要使用洗面乳或洗面皂加強清潔，避免粉刺惡化致發炎、化膿；當症狀惡化時，一定要就醫治療，這時多採用外用藥膏，避免形成痘疤影響容貌，建議口罩應每天更換，並睡眠充足，少吃辛辣食物，維護皮膚健康。

**Comment by Jack-1D** [comment]
對我來說，戴口罩的這幾年真的不太會生病，就只是運動的時候戴著會很悶、很不舒服。即便現在大部分的場合都已經口罩解禁了，我相信還是會有很多人不習慣沒有口罩的生活，而持續戴著口罩。

**Comment by curtis1017** [comment]
最近台灣發生了幾起猴痘事件，而且這幾例猴痘事件，都找不到感染源，甚至從台北到高雄都有，所以猴痘應該已經在台灣默默形成一個傳染鏈，只不過尚未被發現而已。所以我覺得口罩禁令可以再緩緩，因為我們不確定未來會不會有新的傳染疾病。

**Comment by GuanYann** [comment]
戴口罩同時也可避免吸入空氣中的污染物質，例如懸浮微粒、PM2.5等，對肺部和身體健康的影響也不容忽視。因此，即使在戶外，戴口罩仍然是一個非常好的選擇。

**Comment by KevinCheng42** [comment]
我覺得戴口罩的益處大於壞處，因為我容易對濕氣或懸浮微粒等過敏，所以在疫情前就很習慣戴著口罩一整天。雖然戴口罩有時會覺得悶熱，但可以避免傳染病的傳播，也可以降低空氣汙染對我們的影響～

**Comment by zilinng** [comment]
我自己也是過敏人，以前醫生就跟我說想減輕過敏就只能好好戴口罩，但在有口罩令以前我完全戴不住覺得又悶又熱，直到有強制戴口罩的禁令頒布我才慢慢的適應每天都戴口罩的生活，現在雖然禁令解除了但我大部分時間還是會帶著口罩預防過敏，不然整天流鼻水真的超不舒服！

**Comment by CCello** [comment]
我自己到現在還是戴著口罩的，主要是因為目前還沒確診過，至於文章中的皮膚等等問題我會透過勤洗臉來解決，不單可以提振精神，也解決了口罩戴久皮膚會油膩的問題~

**Comment by JessyCheng2003** [comment]
我覺得口罩解禁是好事，這樣我們就可以依自己的意願和狀況，自行決定要不要戴口罩，不用被強制一定要戴口罩。像是化妝的時候，戴口罩不但會很悶還會把臉上的粉用掉，但素顏的時候，戴口罩又可以遮醜～🤣

**Comment by Brownnnn78** [comment]
口罩對台中人一直都不是一個陌生的東西，想當出台中一直沒有太多案例也是不意外，台中會空屋到高中都無法出門上體育課、早上出門搭公車可視距離曾經霧到看不到兩個街口後的招牌，所以從小家裡就總是屯著三大箱(少說有千片)口罩，為了給我這鼻子過敏的可憐鼻涕蟲戴，然後火力發電繼續燒，回台中依然很少拿掉口罩。哭

**Comment by Evening218** [comment]
對於除了吃飯洗澡睡覺以外都戴著眼鏡的我來說,戴口罩本來是一件很痛苦的事情(想想耳朵上面架著眼鏡還掛著口罩,這兩樣東西還很容易就纏在一起哭哭)。但經過這麼長時間的疫情影響,我現在出門如果沒戴口罩真的會覺得不自在,習慣真的很可怕~~~

**Comment by pigeon9975** [comment]
其實到現在我還是習慣戴著口罩，除了安全感外，也可以讓自己遠離各種奇奇怪怪的病毒。
btw最近身邊感冒的人變多了，不知道跟口罩令解除有沒有關係，雖然不能否認夏天戴口罩真的很熱很悶，但如果是在室內場合，戴口罩不但可以保護自己也保護別人，在不會感到不舒服的情況下，其實是個不錯的選擇吧(？

**Comment by orianna901001** [comment]
我現在也是口罩人+1
一想到小時候不用戴口罩到處跑，我覺得是開心又有點驚恐，畢竟1.5公尺內，你光在火車上，都可以吃到其他人的飛沫，現在還是帶著心裡頭總是比較安心。

**Comment by ziihui** [comment]
就算疫情結束後，我也時常帶著口罩。不得不說，戴口罩真的讓我變得很少感冒，記得疫情前一年大概會感冒5、6次，但是自從每天戴口罩後，大幅降低感冒次數，記憶中，這三年來好像也才1、2次(甚至還沒確診過)!

**Comment by Jamesmoneya** [comment]
現在的我出門有四個東西一定要帶錢包鑰始手機口罩，沒有帶口罩的感覺就會有點像是你沒有穿上衣要你去走在街上依樣赤裸，我覺得戴上口罩雖然會增加人跟人之間的距離感，但是這樣的距離我覺得才是我最喜歡的距離

**Comment by ashleyy-huang** [comment]
我是一個非常不喜歡戴口罩的人，尤其是夏天騎摩托車的時候，頂著大太陽戴著安全帽會讓我汗流雨下。不過每當我要搭大眾運輸工具回家的時候我都一定會戴著口罩，因為那樣人來人往的地方不知道淺藏了多少病毒和細菌，戴著一兩個小時可以保持我的個人健康我是很樂意的
